### ISSUE TITLE
False positive for oelint.var.override

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#50]
Snippet

```
COMPATIBLE_MACHINE_armv4 = "(!.*armv4).*"
COMPATIBLE_MACHINE_armv5 = "(!.*armv5).*"
COMPATIBLE_MACHINE_mips64 = "(!.*mips64).*"

COMPATIBLE_HOST_riscv64 = "null"
COMPATIBLE_HOST_riscv32 = "null"
```
